search-app/resources/dependencycheck.properties

```diff
@@ -0,0 +1,4 @@
+analyzer.ruby.gemspec.enabled=false
```

Comment:
Can you add comments explaining these properties? When will we want to set them to `true`?